### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jmontoyaa**
Add missing type + Github definition

### CODE REVIEW

File: packages/auth-js/README.md [hunk 1]
```diff
@@ -22,8 +22,9 @@ AUTH_URL=#optional
 ## How to Use
 
 ```ts
-import { Hono,Context } from 'hono'
-import { authHandler, initAuthConfig, verifyAuth, AuthConfig } from "@hono/auth-js"
+import { Hono, Context } from 'hono'
+import { authHandler, initAuthConfig, verifyAuth, type AuthConfig } from "@hono/auth-js"
+import Google from "@auth/core/providers/google"
```
Comment (yusukebe) [inline packages/auth-js/README.md:27]: Should it be `GitHub`?

**Comment by yusukebe** [comment]
Hi @benjamine 

Thanks! I've left a comment!

File: packages/auth-js/README.md (hunk 1 above)
Comment (jmontoyaa) [reply packages/auth-js/README.md]: Indeed! Fixed

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@jmontoyaa 

Thanks for the quick fix. Merging now.

**Pull request merged**